<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoTimeWindow</class>
 <widget class="QMainWindow" name="GeoTimeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="leftTabButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>GPS and Timing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightTabButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="timeTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>4</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>214</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>26</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
    <row>
     <property name="text">
      <string>Sector 1</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Sector 2</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Sector 3</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Sector 4</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Current</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Best</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
